src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -354,9 +385,16 @@ public CaseDetails submitEventForAttachScannedDocs(
                 e
             );
         } catch (FeignException e) {
+            removeFromIdamCache(e, jurisdiction);
             throw new CcdCallException(
                 String.format("Could not attach documents for case ref: %s Error: %s", caseRef, e.status()), e
             );
         }
     }
+
+    private void removeFromIdamCache(FeignException ex, String jurisdiction) {
+        if (ex.status() == 403 || ex.status() == 401) {
```

Comment:
org.springframework.http.HttpStatus.FORBIDDEN.value(), org.springframework.http.HttpStatus.UNAUTHORIZED.value()

Reply:
I preferred like this as they are numbers